app/src/main/res/values/strings.xml

```diff
@@ -1181,19 +1181,33 @@
     <!-- Onboarding -->
     <!-- Text for onboarding welcome message
     The first parameter is the name of the app (e.g. Firefox Preview) -->
-    <string name="onboarding_header">Welcome to %s!</string>
-    <!-- text for the Firefox account onboarding sign in card header. The word "Firefox" should not be translated -->
-    <string name="onboarding_account_sign_in_header_1">Sync Firefox between devices</string>
-    <!-- Text for the button to learn more about signing in to your Firefox account. The first parameter is the name of the application.-->
-    <string name="onboarding_manual_sign_in_description_2">Bring bookmarks, history, and passwords to %1$s on this device.</string>
+    <string name="onboarding_header" moz:RemovedIn="106" tools:ignore="UnusedResources">Welcome to %s!</string>
+    <!-- Text for onboarding welcome header. -->
+    <string name="new_onboarding_header">Welcome to a better internet!</string>
+    <!-- Text for the onboarding welcome message. -->
+    <string name="onboarding_message">A browser built for people, not profits.</string>
+    <!-- Text for the Firefox account onboarding sign in card header. -->
```

Comment:
Can you keep the original comment for this? No need to update the comment for the old strings that are being moved